ui/app/rif/components/genericSearch/index.js

```diff
@@ -52,13 +56,16 @@ class GenericSearch extends Component {
   }
 
   render () {
-    const {placeholder} = this.props;
+    const {placeholder, onlyFilterOnEnter} = this.props;
+    const handleKeydown = onlyFilterOnEnter ? this.handleInput : null;
+    const handleOnChange = onlyFilterOnEnter ? null : this.handleInput;
     return (
       <div className="search-bar-container">
         <input
           placeholder={placeholder || ''}
           className={'search-bar'}
-          onKeyDown={this.handleKeyDown}
+          onChange={handleOnChange}
```

Comment:
Maybe we could improve this by checking if the input has special characters to skip those to avoid problems.